.gitignore

```diff
@@ -1,8 +1,8 @@
 coverage/
 data
-lib/
+/lib/
 node_modules/
 public/scripts/
 public/styles/
 public/templates/
-spec/
+/spec/
```

Comment:
I think this is not Windows specific, see docs
> A leading slash matches the beginning of the pathname.
For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".

See for similar scenario http://stackoverflow.com/questions/24139478/when-to-use-leading-slash-in-gitignore